ethereum/sourcify
# New UI kaan

Pulling latest changes to the new UI
- Delete `ui-draft/` and rename `ui-new/` to `ui-draft`.
- Deploy the new UI as ui-draft
- Adding new `.env.dev` reference local development env file for easier local setup.
- New `build:clean` npm command to install and build Sourcify.
- Changes to the CheckedContract.ts interface to return the expected and resulting hashes to the UI
- Set CORS to sourcify.dev and subdomains. Also allow localhost with any port if `NODE_ENV=="development"`.
- Add files or zip archives to Verifier with direct URL. Lets adding Github repos via zip download links.
- Design changes to Lookup (prev. Fetcher).
- Fixes Lookup entering infinite loop when a regex valid but checksum invalid address is input.
- Uses ethers.js' `isAddress` function to check valid addresses instead of regex checks.
- Landing page and Verifier design changes.
